content/pages/the-missions/cluster-workshop.md

```diff
@@ -0,0 +1,90 @@
+Title: 
+Save_as: cluster-workshop/index.html
+
+[TOC]
+
+<style>
+ul.schedule > li {
+    margin-top: 0.4em;
+    margin-left: -1em;
+}
+</style>
+
+<img class="img-responsive" style="max-width:100%;" src="../images/k2/bu_fireworks.jpg" alt="Dwarf Stars and Clusters with K2: A Workshop">
+<!-- *Image Credit: Eric Kilby* -->
+
+
+## Summary
+
+The K2 Mission and Boston University will host a **Dwarf Stars and Clusters Workshop on Jan 16-18, 2018**,
+at BU in Boston, Massachusetts. 
+
+Since its launch in 2009, the Kepler telescope has been providing high precision photometric time series on thousands of 
+targets, from exoplanet hosts to active dwarf stars. With its focus on the ecliptic plane, the K2 Mission has further 
+expanded science opportunities, especially in the area of stellar clusters and magnetic activity. By the end of 2017, K2 
+will have observed more than 15 open clusters and associations, some on more than one occasion. The resulting data have 
+enabled ground-breaking research on a number of topics, including gyrochronology, stellar rotation and activity, planet 
+occurence rates, young stellar objects, and eclipsing binary systems. Astronomers working in these areas are warmly 
+invited to gather for discussion at BU this coming January.
+
+The motivations for this workshop include:
+
+* Summarizing the science done to date with K2 in the area of cool stars and open clusters
+* Forging new collaborations for future science with K2
+* Exploring synergies with other space missions (especially TESS), wavelength regions, and science areas.
+
+
+### Agenda
+
+This will be a small workshop with time for most participants to give a short overview of their research during the first 
+two days. (There will not be any posters.) We have reserved the final morning for "hack day" activities, including data how-to's 
+and high-level science product creation overviews. We will also encourage participants to spend time comparing technical 
+approaches to K2 data, including photometry.
+
+<h4 style="font-weight: bold;">Schedule TBA</h4>
```

Comment:
I would already add a coarse schedule here, indicating preliminary time slots for talks, lunch, the hack session, etc.